### DISCUSSION TITLE
YOURLS as a paid service

### DISCUSSION THREAD

**Discussion by jospos** [YOURLS/YOURLS#3681] [General]
Hi!

I would like to make a paid service (WordPress site, Paid Membership Pro and Stripe) and, since I'm not a pro programer - I don't know what is the best way to do it.
Is it a good idea to create a script running once per day, checking which user did not pay and change his url in the database to the "you_did_not_pay" page?
I will hire a freelancer to do coding, but I need to know what task to give to him to do it.

Thank you for any ideas!

**Comment by ehthxC7x9D** [comment]
There are quite a few ways to do it ... and while this may not be the right place for the discussion, I would recommend setting it up with timestamps based on your Stripe authorization callbacks and when/if recurring billing takes place which you could just check programmatically when the paid services portions of the site are requested.   I think you would have more success getting an understanding of the potential flow by reading the Stripe documentation (https://docs.stripe.com/docs/subscriptions) than you would here in this forum.  Ideally your freelancer would have experience integrating with payment API's and could answer those questions for you.
